### PR TITLE
CSI-2413 bump the CRD api version

### PR DESCRIPTION
**Pull request by zingero**
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/csi.ibm.com_ibmblockcsis_crd.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     app.kubernetes.io/instance: ibmblockcsis
     app.kubernetes.io/managed-by: ibm-block-csi-operator
     product: ibm-block-csi-driver
-    release: v1.6.0
+    release: v1.5.0
```
Comment (oriyarde) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/csi.ibm.com_ibmblockcsis_crd.yaml:10]: ?

Comment (zingero) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/csi.ibm.com_ibmblockcsis_crd.yaml]: I reverted that change.

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/csi.ibm.com_ibmblockcsis_crd.yaml]: please update your branch from develop

Comment (zingero) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/csi.ibm.com_ibmblockcsis_crd.yaml]: Done

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

File: deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml [hunk 2]
```diff
@@ -1384,7 +1385,7 @@ spec:
             required:
             - controller
             - node
-            type: object
+          type: object
```
Comment (oriyarde) [inline deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml]: please align your changes to all crd duplications

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/csi.ibm.com_ibmblockcsis_crd.yaml (hunk 1 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/csi.ibm.com_ibmblockcsis_crd.yaml:10]: was this undone?

File: deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml (hunk 2 above)
Comment (zingero) [reply deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml]: Done

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/csi.ibm.com_ibmblockcsis_crd.yaml (hunk 1 above)
Comment (zingero) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/csi.ibm.com_ibmblockcsis_crd.yaml]: Accidentally, yes. now it's ok

**Review by oriyarde — APPROVED** [review]

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**